src/modules/common/components/NavigationTiles.js

```diff
@@ -13,16 +13,12 @@ type PropsType = {|
 |}
 
 const TilesRow = styled.View`
-  margin-top: 20px;
+  background-color: ${props => props.theme.colors.backgroundAccentColor};
   display: flex;
   flex-direction: row;
   align-items: center;
   justify-content: space-around;
-`
-const Separator = styled.View`
-  border-bottom-width: 1px;
-  border-bottom-color: ${props => props.theme.colors.themeColor};
-  margin: 0 15px;
+  elevation: 1;
```

Comment:
Elevation does not work on IOS, we have to set the shadow in a different way there.

Reply:
I added the `shadow-*` properties for iOS with values that I hope are fine. (I couldn't test this).
![image](https://user-images.githubusercontent.com/7025199/57527355-bcd14800-732f-11e9-934c-a5a67bf91e0f.png)